ibis/omniscidb/operations.py

```diff
@@ -313,6 +313,90 @@ def extract_field_formatter(translator, expr, sql_attr=sql_attr):
     return extract_field_formatter
 
 
+def _n_weeks_year(year: ir.NumericValue):
+    """
+    Return the number of the total of weeks for the given year.
+
+    Parameters
+    ----------
+    year : ibis.expr.types.NumericValue
+
+    Returns
+    -------
+    Ibis.expr.types.Expr
+
+    See Also
+    --------
+    https://en.wikipedia.org/wiki/ISO_week_date
+    """
+    # n_weeks_year adjustment
+    def _p(year):
```

Comment:
can you replace `_p` by a descriptive name? Also, the heading `_` is not needed, since this function it's inside another, so it doesn't add much value to mark it as private (it can't be used anywhere else anyway).

Reply:
I used `p` here because it is the name of the function used by the algorithm presented in the link in `See Also` section.
but I can change that.